package.json

```diff
@@ -93,7 +93,7 @@
         "start": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run start ;}; f",
         "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && npm run test:eslint-all && npm run test:jest ;}; f",
         "test:stylelint": "stylelint \"./packages/*/src/**/*.s(a|c)ss\"",
-        "test:jest": "jest --all --maxWorkers=2",
+        "test:jest": "jest --all --maxWorkers=${1:-8}",
```

Comment:
how does this work? :thinking: 
@ali-hosseini-deriv @jim-deriv and I also saw that Ako's suggestion was `${1:8}` but here we have `-8`

Reply:
yes. this is the format: 
[VAR=${[number]:-default}]

thanks man. nice job :+1: